src/components/sections/Screenings.vue

```diff
@@ -33,7 +49,36 @@ export default {
       // making sure something is actually selected
       return this.selected == "" ? this.movies : filteredMovies;
     },
+    currentDayScreenings() {
+      const today = new Date().toDateString();
+      return this.screenings.reduce((todaysScreenings, screening) => {
+        const screeningDate = new Date(screening.datetime);
+        if (screeningDate.toDateString() === today) {
+          todaysScreenings.push({
+            hall: screening.hall,
+            id: screening.id,
+            movie: screening.movie,
+            datetime: screeningDate,
+          });
+        }
+        return todaysScreenings;
+      }, []);
+    },
+    switchScreeningsDays() {
+      const UNIX_ONE_DAY = 100000000; //one day
```

Comment:
how did you get this number? 🤔
I suggest to be more verbose and maybe even keep the calculations in the code (even though it's a constant value) - much easier to find a bug if there is any (btw. like the one you have here, don't you?)

```suggestion
      const DAY_IN_MILISECONDS = 1000 * 60 * 60 * 24
```

Reply:
I was playing around with console and somehow thought it was close enough, until I realized later it's not :P